nuitka/tools/quality/auto_format/YamlFormatter.py

```diff
@@ -0,0 +1,234 @@
+from collections import OrderedDict
+from copy import copy
+from pathlib import Path
+
+import yaml
+
+MASTER_KEYS = [
+    "module-name",
+    "data-files",
+    "dlls",
+    "anti-bloat",
+    "implicit-imports"
+]
+
+DATA_FILES_KEYS = [
+    "dirs",
+    "patterns",
+    "control_tags",
+    "empty_dirs",
+    "empty_dir_structures"
+]
+
+DLLS_KEYS = [
+    "include_from_code",
+    "setup_code",
+    "dll_filename_code",
+    "dest_path",
+    "include_from_filenames",
+    "dir",
+    "patterns"
+]
+
+ANTI_BLOAT_KEYS = [
+    "module_code",
+    "description",
+    "context",
+    "control_tags",
+    "replacements",
+    "replacements_plain",
```

Comment:
As I am adding and changing things here a lot, maybe even structure, it would be nice to take it from the schema file?